### ISSUE TITLE
aot eager accuracy error: DebertaForQuestionAnswering

### ISSUE THREAD

**Issue by desertfire** [pytorch/torchdynamo#1273]
[Edited]
Repro:
```
benchmarks/huggingface.py --training -d cuda --use-eval-mode  --amp  --accuracy --backend aot_eager --only DebertaForQuestionAnswering
```
Error:
```
torchdynamo.utils: [ERROR] RMSE (res-fp64): 5.26979, (ref-fp64): 0.11434
FAIL
```

**Comment by eellison** [comment]
Hmm.. I can't repro this.  Maybe fixed on master

**Comment by eellison** [comment]
Hmm, I think fixed on master. Couldn't repro.

**Comment by desertfire** [comment]
I can still reproduce it. Update cmd as there was some option naming changes.

**Comment by eellison** [comment]
> I tried to turn off fake tensor for this one as well, but got the following error,

Do you mean that was the error without fake tensors ?

**Comment by eellison** [comment]
cc @bdhirsh, this error goes away when you turn off functionalization

> Trying to backward through the graph a second time (or directly access saved tensors after they have already been freed).

**Comment by eellison** [comment]
Fixed with https://github.com/pytorch/pytorch/pull/85658
